startupscript/vertex-ai-user-managed-notebook/post-startup.sh

```diff
@@ -830,18 +830,24 @@ fi
 ######################################
 # Restart proxy to pick up the new env
 ######################################
-APP_PROXY=$(get_metadata_value "instance/attributes/terra-app-proxy")
-readonly APP_PROXY
-TERRA_GCP_NOTEBOOK_RESOURCE_NAME="$(get_metadata_value "instance/attributes/terra-gcp-notebook-resource-name")"
-readonly TERRA_GCP_NOTEBOOK_RESOURCE_NAME
-if [[ -n "${APP_PROXY}" ]]; then
-  emit "Using custom Proxy Agent"
-  RESOURCE_ID=$(get_metadata_value "instance/attributes/terra-resource-id")
-  NEW_PROXY="https://${APP_PROXY}"
-  NEW_PROXY_URL="${RESOURCE_ID}.${APP_PROXY}"
-
-  # Create a systemd service to start the workbench app proxy.
-  cat << EOF > "${WORKBENCH_PROXY_AGENT_SERVICE}"
+IS_NON_GOOGLE_ACCOUNT=$(curl "https://${TERRA_SERVER/verily/terra}-user.api.verily.com/api/profile?path=non_google_account" \
```

Comment:
Is there any plan to add this to the CLI?

It feels like a `wb user status` (without an email address) or `wb user profile` should let me get at some of my basic profile information, including a flag about whether it is a Google-backed account.

Reply:
i agree.. unfortunately we don't have that expose yet. i filed https://verily.atlassian.net/browse/BENCH-2612